plugins/Installation/Controller.php

```diff
@@ -112,15 +111,11 @@ function systemCheck()
             __FUNCTION__
         );
 
-        $view->duringInstall = true;
+        /** @var DiagnosticService $diagnosticService */
+        $diagnosticService = StaticContainer::get('Piwik\Plugins\Diagnostics\DiagnosticService');
+        $view->diagnosticReport = $diagnosticService->runDiagnostics();
```

Comment:
Controllers are created via DI, correct? Can we inject this service, or would that be bad (maybe not performant)?